pkgs/development/python-modules/datasets/default.nix

```
@@ -21,7 +21,7 @@
 
 buildPythonPackage rec {
   pname = "datasets";
-  version = "2.12.0";
+  version = "2.14.4";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/huggingface/datasets/blob/53d55f33bfac9febb0c355e136f2847e5f3e3b53/setup.py#L265

Besides this, LGTM.